pages/mission.js

```diff
@@ -1,106 +1,174 @@
+        <a target="_blank" rel="noreferrer" href={website}>
+        <img
+          src={logo}
+          alt={name}
+        />
+        </a>
+      </div>
+    </div>
   )
 }
 
-export default function Mission() {
+function AwardSection(props) {
+  // title, provider
   return (
-    <main role="main">
-      <br></br>
-      <section className="container u-content-space u-bg-arrow-wrapper">
-        <div className="text-center u-bg-arrow-bottom">
-          <p className="h5 mb-0">ICS Student Council was created by <strong className="text-primary">passionate </strong> students for all interested students.</p>
-        </div>
-      </section>
+    <Col style={{textAlign:"center", color:"black", maxWidth:500, marginTop:25, marginBottom:25}} md={6}>
+      <h4>{props.title}</h4>
+      <p>{props.provider}</p>
+    </Col>
+  )
+}
 
-      <section className="text-center bg-light u-content-space">
-        <div className="container">
-          <header className="text-center w-md-50 mx-auto mb-8">
-            <h2 className="h1">Our Mission</h2>
-          </header>
+function Person(props) {
+  // name, position, email, image
+  return (
+      <div className={styles.person}>
+        <img alt={props.image} src={props.image} alt={props.name}/>
+        <h5>{props.position}</h5>
+        <h4>{props.name}</h4>
+        <p>{props.email}</p>
+      </div>
+  );
+}
 
-          <Row>
-            <Col lg={6} className="mb-5">
-              <div className="display-4 text-primary mb-4">
-                <FaRegLightbulb />
-              </div>
-              <h4 className="h5">Improve</h4>
-              <p className="mb-0">Student life of the Donald Bren School of Information and Computer Sciences academically, socially, and professionally.</p>
-            </Col>
+function PastBoard(props) {
+  const { year, positions } = props;
+  return (
+    <div className={styles.pastSection}>
+      <h4>{year}</h4>
+      <Row className="justify-content-center">
+        {positions.map(position => {
+          const {title, name} = position
+          return (
+            <div className={styles.pastBoard}>
+              <h5>{name}</h5>
+              <p>{title}</p>
+            </div>
+            )
+          })}
+      </Row>
+    </div>
+  )
+}
 
-            <Col lg={6} className="mb-5">
-              <div className="display-4 text-primary mb-4">
-                <FaRegHandshake />
-              </div>
-              <h4 className="h5">Create and Deepen</h4>
-              <p className="mb-0">Bonds between students, faculty, administration, companies, and alumni of the ICS School as well as similar groups.</p>
-            </Col>
+export default function Mission() {
+  return (
+    <>
+      {/* Mission Section */}
+      <div className="sectionAlt">
+        <h2>Our Mission Statement</h2>
+        <p>Improve student life of the Donald Bren School of Information and Computer Sciences academically, socially, and professionally 
```

Comment:
> Donald Bren School of Information and Computer Science**s**

I was at UCI for 5 years and had no clue that there was an "**s**". I always thought it was just "Computer Science" lol